packages/shopify-app-express/src/__tests__/index.test.ts

```diff
@@ -36,6 +36,32 @@ describe('shopifyApp', () => {
   });
 
   it('properly defaults missing configs based on env vars', () => {
+    /* eslint-disable no-process-env */
+    process.env.SHOPIFY_APP_API_KEY = 'envKey';
+    process.env.SHOPIFY_APP_API_SECRET = 'envSecret';
+    process.env.SHOPIFY_APP_SCOPES = 'envScope1,envScope2';
+    process.env.SHOPIFY_APP_URL = 'https://envHost';
+    process.env.SHOP_CUSTOM_DOMAIN = '*.envCustomDomain';
```

Comment:
I assume we provide some docs on the package so partners can know what env vars to provide.  Do we need to update these too?

Reply:
We're not using env vars in the github docs, they're just the defaults. We just use strings for these values [in the docs](https://github.com/Shopify/shopify-app-js/tree/main/packages/shopify-app-express#getting-started).

Reply:
Ok, I still find the use of env vars in the package a bit confusing, but I think that's one for another time.